flume/doc/design/architecture.md

```diff
@@ -35,10 +35,11 @@ You can simply add more sinks consuming events from the same single channel. Thi
 ### Multiple sinks, multiple channels
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-connectors/commit/baebd868b650642fec8557b63778e1b18a5020b3